src/sentry/api/event_search.py

```diff
@@ -141,6 +141,9 @@ def translate(pat):
     'first_seen': 'first_seen',
     'last_seen': 'last_seen',
     'times_seen': 'times_seen',
+    # OrganizationEvents aggregations
+    'event_count': 'event_count',
+    'user_count': 'user_count',
```

Comment:
Will this impact regular event search in any way?

Reply:
Hi @lynnagara . Yes if the user entered in the exact filter keys of `event_count` or `user_count` then it would attempt to search for those things but not have the defined aggregations for them. I think the chance of that is very low. But I can follow up with a pr handling that case. I was going to take on the task about better error handling in event search anyway `SEN-727`. Good catch.